src/controllers/user.js

```diff
@@ -54,5 +54,30 @@ class UserController {
       res.status(400).json({ error: error.message });
     }
   };
+
+  static assignRoles = async (req, res) => {
+    const userRole = req.body.RoleId;
+    const userExist = await User.findOne({
+      where: { email: req.body.Email },
```

Comment:
remember if someone signed up with Facebook he does not have email. how will admin assign role to this user? i think you can use id instead of email

Reply:
> remember if someone signed up with Facebook he does not have email. how will admin assign role to this user? i think you can use id instead of email

@Robertishimwe The Task says that the role is suppossed to be assigned to the user using an email so I think there is a separate one for that me I am just following the task description.

Reply:
Ok

Reply:
Please do as @Robertishimwe suggests.